sites/website/src/docs/community/engineering-process/issue-management.md

```diff
@@ -0,0 +1,120 @@
+---
+id: issue-management
+title: Issue Management
+sidebar_label: Issue Management
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/engineering/issue-management.md
+---
+Managing a popular GitHub repo with a small team is not an easy task. It requires a good balance between creating new features while handling many investigations and bug fixes associated with existing ones.
+
+During the last couple of years the amount of incoming issues has been constantly growing. While this is a sign of a healthy framework and ecosystem surrounding it, it's becoming harder to react to all those issues.
+
+To keep up with ever-evolving expectations, we use a set of rules to help us better handle the incoming issues.
+
+## Goals
+
+The goals of these rules are listed below in priority order:
+
+- Make it easy to make triage decisions for each issue filed in this repository
+- Be able to easily prioritize issues
+- Set proper expectations with customers regarding how issues are going to be handled
+
+## Triage Process Details
+
+The feature teams should be able look through every single issue filed against this repository and be able to make a quick call regarding the nature of the issue.
+We will first categorize the issues and further handle these depending on the category the issue is in. The subsections below represent these categories and the rules we apply for them during our triage review.
+
+### Information Gathering
+
+In this phase we instruct the user on how to collect the appropriate diagnostics and see if they are able to address the issue with that additional information.  When we need user input we will mark the issue with the `status:needs-information` label. Issues in this phase may be closed automatically if we do not receive timely responses; they often do not provide enough information for us to investigate further.
```

Comment:
You might consider linking to the specific policy for `status:needs-information` here. My first thought in reading this was "what constitutes a 'timely response'"

Reply:
I consider a timely response to be less than the amount of time you would grow frustrated if you were blocked on an issue that you filed into some system.  For example, I file a lot of bugs and if they were not triaged in 24 hours for open source I would be disappointed.  So, IMHO I believe 12 hours is a timely response.  The faster we respond to our community the more they will praise us, and thus share their experience.  Especially, for first-timers, and newbies.